react/src/components/QuotaPerStorageVolumePanelCard.tsx

```diff
@@ -0,0 +1,229 @@
+          onChange={(__, vInfo) => {
+            setSelectedVolumeInfo(vInfo);
+          }}
+          autoSelectType="usage"
+          showUsageStatus
+          showSearch
+          allowClear
+        />
+      }
+      styles={{
+        body: {
+          paddingTop: token.paddingLG,
+        },
+      }}
+    >
+      {selectedVolumeInfo !== deferredSelectedVolumeInfo ? (
+        <FlexActivityIndicator style={{ minHeight: 120 }} />
+      ) : selectedVolumeInfo?.capabilities?.includes('quota') ? (
+        <Row gutter={[24, 16]}>
+          <Col
+            span={12}
+            style={{
+              borderRight: `1px solid ${token.colorBorderSecondary}`,
+            }}
+          >
+            <BAIProgress
+              title={
+                <Flex direction="column" align="start">
+                  <Typography.Text
+                    type="secondary"
+                    style={{ fontSize: token.fontSizeSM }}
+                  >
+                    {t('data.Project')}
+                  </Typography.Text>
+                  <Typography.Text style={{ fontSize: token.fontSize }}>
+                    {currentProject?.name}
+                  </Typography.Text>
+                </Flex>
+              }
+              percent={projectPercent}
+              used={
+                _.isEmpty(projectUsageBytes)
+                  ? ''
+                  : `${convertDecimalSizeUnit(_.toString(projectUsageBytes), 'G')?.numberUnit}B`
+              }
+              total={
+                _.isEmpty(projectHardLimitBytes)
+                  ? ''
+                  : `${convertDecimalSizeUnit(_.toString(projectHardLimitBytes), 'G')?.numberUnit}B`
+              }
+            />
+          </Col>
+          <Col span={12}>
+            <BAIProgress
+              percent={userPercent}
+              title={
+                <Flex direction="column" align="start">
+                  <Typography.Text
+                    type="secondary"
+                    style={{ fontSize: token.fontSizeSM }}
+                  >
+                    {t('data.User')}
+                  </Typography.Text>
+                  <Typography.Text style={{ fontSize: token.fontSize }}>
+                    {baiClient?.full_name}
+                  </Typography.Text>
+                </Flex>
+              }
+              used={
+                _.isEmpty(userUsageBytes)
+                  ? ''
+                  : `${convertDecimalSizeUnit(_.toString(userUsageBytes), 'G')?.numberUnit}B`
+              }
+              total={
+                _.isEmpty(userHardLimitBytes)
+                  ? ''
+                  : `${convertDecimalSizeUnit(_.toString(userHardLimitBytes), 'G')?.numberUnit}B`
+              }
+            />
+          </Col>
+        </Row>
+      ) : (
+        <Empty
+          image={Empty.PRESENTED_IMAGE_SIMPLE}
+          description={t('storageHost.QuotaDoesNotSupported')}
+          style={{ margin: '25px auto' }}
```

Comment:
The empty part of the screen seems to be skewed downward. How about changing it to something like this?

```suggestion
          style={{ margin: 'auto 25px' }}
```